frontend/app/views/spree/products/_filters_desktop.html.erb

```diff
@@ -47,7 +47,7 @@
             <% price_param = params[:price] == name ? '' : name %>
             <%= link_to permitted_params.merge({ price: price_param }), class: "d-block" do %>
               <div class="d-inline-block py-1 px-2 m-1 text-uppercase plp-filters-card-item <%= 'plp-filters-card-item--selected' if params[:price] == name %>">
-                <%= name %>
+                <%= name.upcase %>
```

Comment:
```suggestion
                <%= name%>
```

@Punkbooster we don't need to do this, there's already `text-uppercase` CSS class attached